accepted/PSR-16-simple-cache.md

```diff
@@ -149,13 +149,13 @@ interface CacheInterface
      * @throws \Psr\SimpleCache\InvalidArgumentException
      *   MUST be thrown if the $key string is not a legal value.
      */
-    public function get($key, $default = null);
+    public function get(string $key, mixed $default = null): mixed;
```

Comment:
Consistent with the other specs, the spec text itself *should not be changed* with types.  Just update the code package itself.